### PR TITLE
Feature/1 Extracted from product-integration-tool-chest

### LINKED ISSUE #1: Extract from product-integration-tool-chest
Extract github-keeper from product-integration-tool-chest

### PR DESCRIPTION
**Pull request by jakobbraun**
Closes #1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
+# GitHub Keeper
+
+GitHub keeper is a CLI tool that helps to unify our repositories.
+
+Features:
+
+* List all Exasol repos where you are admin (`list-my-repos`)
+* Create / Update the branch protection for the default branch according to our standards (`create-branch-protection`)
+
+#### Installation
```
Comment (morazow) [inline README.md:10]:
```suggestion
## Installation
```

File: README.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+1. Install Go language. Minimum version is 1.13. 
+2. Install dependendent packages: 
+```
+cd $HOME/<PATH TO THIS REPO>/product-integration-tool-chest/github-keeper
+go get ./...
+```
+
+#### Usage
```
Comment (morazow) [inline README.md:19]:
```suggestion
## Usage
```

Comment (morazow) [reply README.md:19]: Maybe good idea to follow first, second then third level headings.

**Review by morazow — DISMISSED** [review]
Approved with minor suggestion

File: README.md (hunk 2 above)
Comment (jakobbraun) [reply README.md]: yes. Copy and paste issue...

**Review by morazow — APPROVED** [review]

**Pull request merged**